spec/_attachments/ic.did

```diff
@@ -184,6 +192,14 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // metrics interface
+  metrics : (record {
+    subnet_id : principal;
+  }) -> (record {
+    timestamp_nanos : nat64;
```

Comment:
I've now drafted the API to return a single timestamp in the response that should characterize all returned metrics across all nodes. I believe this makes sense given that the metrics are gonna be fetched from the replicated state and the replicated state has a unique time associated with it (see function `ReplicatedState::time`).

Reply:
It's not clear from the type or the description whether the API will return one `metrics` entry for every subnet node, multiple entries per subnet node; or even multiple entries for a single node. I'm guessing it's the former (even though that conflicts with what David was saying in the other comment thread), but it's never spelled out.

Reply:
@mraszyk, I agree that one timestamp is sufficient for this. 

@alin-at-dfinity: we haven't fully converged yet on what should be returned (we discussed several options and their pros and cons on Friday with Martin, David Derler, Dimitris and Sasa).
Once we know, we should definitely make it explicit.

For simpler auditability, multiple entries per subnet subnet node would be easiest, I'll try to expand on that in the other open thread.